File: src/entity/enemy/SpeedEnemy.java
package entity.enemy;

import javafx.scene.image.Image;

public class SpeedEnemy extends Enemy {
	
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile248.png";
	
	public SpeedEnemy(int x, int y) {
		super(x, y, 5, 3);
		img = new Image(IMAGE_URL);
		setDir('E');
	}
	
}

File: src/entity/enemy/SpeedEnemy.kt
package entity.enemy

import javafx.scene.image.Image

class SpeedEnemy(x: Int, y: Int) : Enemy(x.toDouble(), y.toDouble(), 5, 3.0) {

    init {
        img = Image(IMAGE_URL)
        setDir('E')
        reward = 3
    }

    companion object {

        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile248.png"
    }

}
